litho-core/src/main/java/com/facebook/litho/ComponentTree.java

```diff
@@ -1952,6 +2039,170 @@ protected LayoutState calculateLayoutState(
     }
   }
 
+  private class LayoutStateFuture extends FutureTask<LayoutState> {
+
+    private static final int THREAD_PRIORITY_INVALID = Process.THREAD_PRIORITY_LOWEST +
```

Comment:
Maybe just use Integer.MAX_INT?

Reply:
I was thinking about that, but concerned someone may try to increment it and cause it to wrap around and we'd loop through a crazy number of integers before it stops throwing SecurityExceptions, but maybe I'm just being paranoid.

Reply:
removed in favor of boolean